src/commands/commit/list.ts

```diff
@@ -88,7 +88,8 @@ export default class CommitList extends BaseCommand<typeof CommitList> {
       },
       commit_message: {
         header: "Commit message",
-        get: (entry) => entry.commit_message?.trim(),
+        get: (entry) =>
+          entry.commit_message ? entry.commit_message.trim() : "",
```

Comment:
Sometimes there are new lines after a given entry, which looks really bad in the terminal. After digging into this, I realized the solution is to delete all whitespaces in the commit message

Before:

<img width="1440" alt="Screen Shot 2023-11-07 at 13 49 35" src="https://github.com/knocklabs/knock-cli/assets/67347884/7c9656ba-faf1-4286-a2c6-323991bf01b2">

After:
<img width="1439" alt="Screen Shot 2023-11-07 at 13 49 53" src="https://github.com/knocklabs/knock-cli/assets/67347884/10b1852d-55d2-4ac7-aae2-a1b06c02449e">

Reply:
Yep that makes sense. 

One related question though — what happens with a really long commit message? In the screenshots above, it looks like we are doing truncation and I'm guessing that's automatically done by oclif?

Reply:
Yes initially I added a custom truncation function, but I removed it because oclif already uses one. And it works very well btw.

Reply:
> And it works very well, the table takes all screen space available and the message truncates if it does not fit in the column.

Got it, thanks for clarifying. One issue here though is then, what happens if you want to see the entire identifier or the commit message without getting truncated? Just thinking out loud, we may need the `commit get` command for this?

Reply:
Good catch, we can do that in another PR if we feel like it.

Reply:
> Good catch, we can do that in another PR if we feel like it.

Yes that works!

Reply:
Btw, I don't think this should block the release of this feature, and is something we can add later. But we can discuss more of this with the team if you prefer.

Reply:
If I were to make a call I personally would get that in before releasing the feature because 1) it's straight forward enough, but also because 2) right now the commit message is the only way to reason about what it is that you are trying to promote and not being able to see the entire message feels a hindrance to using the feature. 

Maybe there's a way to force oclif not to truncate and print out the whole commit message but then that may introduce another issue of the table formatting getting messed up. 

Anyways open to discussion for sure!

Reply:
Ok I said this because this feature was planned to be shipped in the next changelog tomorrow, as we thought there was not going to be more changes. I can try to get it done before that, but with that + reviewing it (and we are still in code review in other PR'S) we probably don't get to the timeline.

I can propose moving this to next changelog as well.

Reply:
Gotcha — yeah let's focus on getting the current outstanding PRs turned around and merged, and then we can chat about the changelog and when it goes out.